### PR TITLE
New package: vagrant-1.7.2

### PR DESCRIPTION
**Pull request by thypon**
#1234

### CODE REVIEW

**Comment by thypon** [comment]
The package it's a bundle containing all the ruby libraries needed to run vagrant.

@xtreame there is no simple way to install from rubygems. The rubygems way of packaging is deprecated by the author.

File: srcpkgs/vagrant/template [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Template file for 'vagrant'
+pkgname=vagrant
+version=1.7.2
+revision=1
+hostmakedepends="ruby"
+makedepends="ruby-devel zlib-devel"
+depends="rtmpdump curl"
```
Comment (beefcurtains) [inline srcpkgs/vagrant/template]: rtmpdump??? Really???

**Comment by xtraeme** [comment]
This pkg is installing multiple random files that don't belong to the final binary package, `.c, .log` etc.

Won't be accepted

**Comment by thypon** [comment]
Fixing them.

**Comment by thypon** [comment]
Maybe a little to clean.

**Comment by xtraeme** [comment]
These files:

```
/home/travis/build/voidlinux/void-packages/masterdir/destdir//vagrant-1.7.2/usr/lib/vagrant/extensions/x86_64-linux/2.2.0/nokogiri-1.6.3.1/gem_make.out

/home/travis/build/voidlinux/void-packages/masterdir/destdir//vagrant-1.7.2/usr/lib/vagrant/extensions/x86_64-linux/2.2.0/nokogiri-1.6.3.1/gem.build_complete
```

afaik are not really necessary.

**Comment by thypon** [comment]
Merged manually + updated

**Pull request closed without merging**